code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -157,17 +188,29 @@
 			owner.hud_used.move_intent.icon_state = "running"
 		owner.update_icons()
 
-	cancel_stealth()
+	if(pounce_break_stealth)
+		cancel_stealth()
 
 /// Callback for when a mob gets hit as part of a pounce
 /datum/action/ability/xeno_action/stealth/proc/mob_hit(datum/source, mob/living/M)
 	SIGNAL_HANDLER
 	if(M.stat || isxeno(M))
 		return
 	if(can_sneak_attack)
-		M.adjust_stagger(3 SECONDS)
-		M.add_slowdown(1)
+		var/datum/action/ability/activable/xeno/hunter_mark/assassin/mark = owner.actions_by_path[/datum/action/ability/activable/xeno/hunter_mark/assassin]
+		if(mark.marked_target == M)
+			to_chat(owner, span_xenodanger("We strike our death mark with a calculated pounce."))
+			M.adjust_stagger(6 SECONDS)
+			M.add_slowdown(2)
+		else
+			M.adjust_stagger(3 SECONDS)
+			M.add_slowdown(1)
 		to_chat(owner, span_xenodanger("Pouncing from the shadows, we stagger our victim."))
+	if(pounce_hit_break_stealth)
+		cancel_stealth()
+	else
```

Comment:
just early return

Reply:
iirc doing that removed the sneak attack cooldown after you first achieved it, you could slap it each slash, doesnt work.